packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -186,6 +184,28 @@ export const RemoteServerFactory = {
         // Hook for the dev-server
     },
 
+    /**
+     * Passing the requestId from apiGateway event to express request object
+     * @private
+     */
+    _addRequestIdToReq(app) {
+        app.use((req, res, next) => {
+            if (!req.headers['x-apigateway-event']) {
+                console.error('Missing x-apigateway-event')
+                next()
+                return
+            }
+            const apiGatewayEvent = JSON.parse(
+                decodeURIComponent(req.headers['x-apigateway-event'])
+            )
+            const {requestId} = apiGatewayEvent.requestContext
+            req['apiGateway'] = {
+                requestId
+            }
+            next()
+        })
+    },
```

Comment:
Couple comments: 

1. Why is this an inline middleware when the version of the dev server is in its own file, wouldn't it be better to be consistent?
2. The name of this function add request id to request.. but it looks like you are adding a not adding the request id but an object call api gateway with an id in it. Do we want to be a little more declarative in what we are doing? You can go down 2 routes, first would be call the function addAPIGatewayContext or something like that, where for now we only add the request id to it with room to grow. Or you just add the request id without the wrapper. 
3. Related to the second option in number 2 above, could we not directly add the requestId to the locals? That would mean we dont even need `that._setRequestId`